tools/rtrclient.c

```diff
@@ -106,7 +106,7 @@ static void update_spki(struct spki_table* s __attribute__((unused)), const stru
 
 int main(int argc, char** argv)
 {
-    enum mode_t { TCP, SSH } mode;
+    enum mode_t { NOP, TCP, SSH } mode = NOP;
```

Comment:
It probably fixes a use before initialization warning?
I think it would be better to add an else statement in line 130 and terminate the client here if argv[1] contains a different String than "tcp" or "ssh"